elliottlib/cli/find_bugs_sweep_cli.py

```diff
@@ -314,14 +312,19 @@ def categorize_bugs_by_type(bugs: List[Bug], advisory_id_map: Dict[str, int], ma
     not_found = set(tracker_bugs) - found
     if not_found:
         not_found_with_component = [(b.id, b.whiteboard_component) for b in not_found]
-        red_prefix("Tracker Bugs Warning: ")
-        click.echo("The following (tracker bug, package) were found BUT not attached,"
-                   " since no corresponding brew build was found attached to any advisory. "
-                   "First attach builds to the correct advisory and rerun to attach the bugs, "
-                   "or exclude the bug ids in the assembly definition")
-        click.echo(not_found_with_component)
-        raise ValueError(f'No builds found for CVE (bug, package): {not_found_with_component}. Either attach '
-                         f'builds or exclude the bugs in the assembly definition')
+        # Do not error out if a single advisory is requested to be operated on
+        if not default_advisory_type:
```

Comment:
This doesn't seem to be correct. If `--into-default-advisories` flag is set, `default_advisory_type` will be `None` as well and may trigger an error. I think the logic to handle different flag combinations needs to be refactored:
- When `default_advisory_type` is set.
- When `--into-default-advisories` is set.
- When `advisory_id` is set.
- None of above is set.

Reply:
Ah I didn't account for when advisory_id is set. The logic is getting pretty hairy here, accounting for all the 
options. What do you think about deprecating the advisory_id option? Maybe not here but in another PR - I don't think it gets used much - and we have attach-bugs for other cases

Reply:
Maybe this can be refactored like this:
- In `categorize_bugs_by_type`, remove `default_advisory_type` parameter.
- In the caller function, only pass the advisories we'd like to sweep through `advisory_id_map` parameter. e.g. if  `default_advisory_type` is set, we pass `{default_advisory_type: advisory_id}`.
- If `advisory_id` is set. we don't call `categorize_bugs_by_type` at all so no error will be raised.
What do you think?